src/BaGet.Core/Indexing/SymbolIndexingService.cs

```diff
@@ -158,7 +188,7 @@ bool IsValidSymbolFileInfo(FileInfo file)
                         signature = id.Guid.ToString("N").ToUpperInvariant();
                     }
 
-                    var fileName = Path.GetFileName(pdbPath).ToLowerInvariant();
+                    var fileName = Path.GetFileName(pdbPath)?.ToLowerInvariant();
```

Comment:
I added this because we have no assertion in this method that string parameter won't be null - makes the method little more defensive. I can remove it if you think it has negative impact.